Bio/Align/bigbed.py

```diff
@@ -75,6 +75,8 @@
 class AutoSQLTable(list):
     """AutoSQL table describing the columns of an (possibly extended) BED format."""
 
+    default: "AutoSQLTable"
```

Comment:
What does this do? Is it interpreted as a type hint by mypy (and other type aware systems)?

Reply:
Yes it annotates `AutoSQLTable.default` as type `AutoSQLTable`. I you remove it, mypy reports:
```
Bio/Align/bigbed.py:167:1: error: "type[AutoSQLTable]" has no attribute "default"  [attr-defined]
    AutoSQLTable.default = AutoSQLTable(
    ^~~~~~~~~~~~~~~~~~~~
```